### DISCUSSION TITLE
Console version

### DISCUSSION THREAD

**Discussion by gumbymoegoe** [SebLague/Geographical-Adventures#78] [Ideas]
This would be absolutely amazing on an Xbox / Playstation
would it be possible to make such a version?

keep up the great work

**Comment by arrowgent** [comment]
yes but applying for license to those platforms is incredibly high and delivering updates is a lengthy process with additional cost

consider $100,000 to get on the platform then $5,000-$25,000 for each update
by comparison Steam is only $100.  and Itchio is free.

Playstation license requires "trophies" in the game to get on the store.  another headache which isnt required for a game but arbitrary feature forced by sony.

**PC is basically free, with zero additional cost for releasing the game**
